### DISCUSSION TITLE
self-hosting/federation-2

### DISCUSSION THREAD

**Discussion by giscus** [graphql-hive/console#1780] [Docs Discussions]
# self-hosting/federation-2

GraphQL Hive Documentation

https://docs.graphql-hive.com/self-hosting/federation-2

**Comment by pierrebiver** [comment]
Hello,

you made a mistake in your example where you use `_` for defining the service name, but use `-` when declaring the service in the UI. It took me a day to figure it out because the log were not explicit, it would be nice to fix it so people won't make the same mistake :)

**Reply by dotansimha** [reply]
Thank you @pierrebiver , I fixed that in https://github.com/kamilkisiela/graphql-hive/pull/1789
